lib/src/utils.dart

```diff
@@ -0,0 +1,83 @@
+/// Returns a path from a given `gs://` URL.
```

Comment:
If this comes verbatim from another repository, it is good practice to link it at the beginning of the file. This way if the original repository modifies the logic, we know where to import it from again.

Looks like it came from https://github.com/firebase/flutterfire/blob/790e450e8d6acd2fc50e0232c77a152430c7b3ea/packages/firebase_storage/firebase_storage/lib/src/utils.dart.